### PR TITLE
Feature/cleaning

### PR DESCRIPTION
**Pull request by harshvitra**
### Changes in PR - 

**Global Changes -** 
1. Enabled pull to refresh - global.scss
2. Removed react-js-pull-to-refresh library

**App.tsx -** 
1. Installed @types/react-gtm-module
2. Changed frame.filename to frame.filename?, removes undefined error
3. Removed comments -   // const { useTranslation } = i18next;  // const { i18n } = useTranslation();
4. Removed - language gets detected by i18next-browser-languagedetector
5. Removed - console.log(process.env.NEXT_PUBLIC_GA_TRACKING_ID);

**Leaderboard page -**
1. Removed leaderboard page completely

**Home Page -** 
1. Removed meta tags 
2. Removed Pull to refresh code
3. Removed onRefresh function
4. Removed isMobile code
5. Shifted API calls

**404 Page -** 
1. Shifted styles 

**Index file -** 
1. Removed - import { route } from 'next/dist/next-server/server/router';
2. Removed console log of use effect
3. Removed logic for fetching APIs and created new functions
4. Removed Meta tags from index file and created 2 new functions for this

**TreeCounter (Common)**
1. Removed the hidetarget code
2. Centered the design

**User Profile page**
1. Shifted Layout to show Footer and Navbar while loading
2. Created API for fetching user profile
3. Shifted Meta tags

**Public folder**
1. Remove unused image from Home folder
2. Delete leaderboard png files and shift svgs to planet folder
3. Removed Roadwebp.webp (unused)
4. Removed various unused icons

**src/assets folder**
1. Removed unused icons
2. Shifted all files to public/assets

**src/common**
1. Shifted Layout files to Layout folder

**Created -** 

- getStoredCurrency - to get the stored or the default currency
- Created Meta tags functions for index and project page
- Created Meta tags functions for home page
- Created meta tags function for user profile
- Created common get api function

### CODE REVIEW

**Comment by harshvitra** [comment]
Because of the Logo size of Stern at 330px and below the bottom navbar is disturbed
Screenshot 2020-10-07 at 5 13 43 PM

File: src/utils/apiRequests/getAllProjects.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import getsessionId from './getSessionId';
+import getStoredCurrency from '../countryCurrency/getStoredCurrency';
+
+export async function getAllProjects() {
```
Comment (norbertschuler) [inline src/utils/apiRequests/getAllProjects.ts:4]: All these API call functions are having mostly the same code which can be abstracted.

Comment (harshvitra) [reply src/utils/apiRequests/getAllProjects.ts:4]: Ideally when all APIs are in place we will abstract this code to create one function for get, post, put requests accordingly

Comment (norbertschuler) [reply src/utils/apiRequests/getAllProjects.ts:4]: I already build an abstraction for you to check at https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/297

Comment (harshvitra) [reply src/utils/apiRequests/getAllProjects.ts]: Thanks a ton !

**Pull request merged**